docs/theme/static/js/components/upcoming-event/upcoming-event.js

```diff
@@ -0,0 +1,71 @@
+/* global moment */
+import $ from 'jquery';
+// import moment from 'moment'; // added globally
```

Comment:
I see now that this was how it was done already. Feel free to leave it like this.